extensions/devservices/common/src/main/resources/devservices.properties

```diff
@@ -0,0 +1 @@
+mariadb.image=${mariadb.image}
```

Comment:
So you would have one common file for all the images, right?

I thought of doing it with a file specific to each devservice but it's certainly easier this way and good enough IMHO. This could be generalized to all devservices next.

Reply:
@gsmet Yeah, thought about multiple files too. I can change it to a specific one per dev service if you prefer.
This would make loading that specific file a bit more complicated though, but it's definitely doable.

Reply:
PS: This would also make https://github.com/quarkusio/quarkus/pull/23329#discussion_r796132380 obsolete.

Reply:
FWIW:
> This could be generalized to all devservices next.

I have started working on this.